index.html

```diff
@@ -2733,6 +2733,31 @@ <h2 id="OfflineAudioCompletionEvent">
           </dd>
         </dl>
         <section>
+          <h2>
+            Dictionaries
+          </h2>
+          <dl title="dictionary AudioNodeChannelOptions" class="idl">
```

Comment:
I think this should be called `AudioNodeOptions`. There was no need to define an additional dictionary for AudioNodes for the number of inputs and outputs because these are never settable for any node except for the merger, splitter, and script processor.  I thought it was better just to add the appropriate input/output number separately for each of these nodes.